bundles/org.openhab.core.ui/src/main/java/org/openhab/core/ui/internal/components/UIComponentSitemapProvider.java

```diff
@@ -90,251 +77,172 @@
+                    break;
+                case Webview webviewWidget:
+                    setWidgetPropertyFromComponentConfig(webviewWidget, component, "height");
+                    setWidgetPropertyFromComponentConfig(webviewWidget, component, "url");
+                    break;
+                case Mapview mapviewWidget:
+                    setWidgetPropertyFromComponentConfig(mapviewWidget, component, "height");
+                    break;
+                case Slider sliderWidget:
+                    setWidgetPropertyFromComponentConfig(sliderWidget, component, "minValue");
+                    setWidgetPropertyFromComponentConfig(sliderWidget, component, "maxValue");
+                    setWidgetPropertyFromComponentConfig(sliderWidget, component, "step");
+                    setWidgetPropertyFromComponentConfig(sliderWidget, component, "switchEnabled");
+                    setWidgetPropertyFromComponentConfig(sliderWidget, component, "releaseOnly");
+                    break;
+                case Selection selectionWidget:
+                    addWidgetMappings(selectionWidget.getMappings(), component);
+                    break;
+                case Input inputWidget:
+                    setWidgetPropertyFromComponentConfig(inputWidget, component, "inputHint");
+                    break;
+                case Setpoint setpointWidget:
+                    setWidgetPropertyFromComponentConfig(setpointWidget, component, "minValue");
+                    setWidgetPropertyFromComponentConfig(setpointWidget, component, "maxValue");
+                    setWidgetPropertyFromComponentConfig(setpointWidget, component, "step");
+                    break;
+                case Colortemperaturepicker colortemperaturepickerWidget:
+                    setWidgetPropertyFromComponentConfig(colortemperaturepickerWidget, component, "minValue");
+                    setWidgetPropertyFromComponentConfig(colortemperaturepickerWidget, component, "maxValue");
+                    break;
+                case Buttongrid buttongridWidget:
+                    addWidgetButtons(buttongridWidget.getButtons(), component);
+                    break;
+                case Button buttonWidget:
+                    setWidgetPropertyFromComponentConfig(buttonWidget, component, "row");
+                    setWidgetPropertyFromComponentConfig(buttonWidget, component, "column");
+                    setWidgetPropertyFromComponentConfig(buttonWidget, component, "stateless");
+                    setWidgetPropertyFromComponentConfig(buttonWidget, component, "cmd");
+                    setWidgetPropertyFromComponentConfig(buttonWidget, component, "releaseCmd");
+                    break;
+                case Default defaultWidget:
+                    setWidgetPropertyFromComponentConfig(defaultWidget, component, "height");
+                    break;
+                default:
```

Comment:
Looks like you forgot the case of Switch to call addWidgetMappings

Reply:
Indeed, thank you.

Reply:
Resolved